service/src/integrationTest/java/uk/gov/hmcts/reform/fpl/controllers/CaseInitiationMidEventControllerTest.java

```diff
@@ -47,6 +48,9 @@
     @MockBean
     private OrganisationService organisationService;
 
+    @MockBean
+    private AddCaseAssignedUserRolesRequest addCaseAssignedUserRolesRequest;
```

Comment:
not needed

Reply:
sorted

Reply:
not resolved, this file should not have any changes for this story